src/pages/about.tsx

```diff
@@ -193,6 +193,15 @@ export default function About() {
               code of conduct (below) and the styleguide (in the Figma)
               carefully before contributing.
             </p>
+            <p className={styles.aboutDev}>
+              <strong>Improve the website</strong>
+              <br />
```

Comment:
Doesn't need to block this PR, but I'd' make these sub-headings (`<h3>`).

Also, I think these icons look great, and would look even better if they were a touch lighter than the text (like 0.75 opacity or something like that).